dxr/lines.py

```diff
@@ -331,7 +296,7 @@ def finished_tags(lines, refs, regions):
     # Plugins return unicode offsets, not byte ones.
 
     # Get start and endpoints of intervals:
-    tags = list(tag_boundaries(refs, regions))
+    tags = list(tag_boundaries(list(chain(refs, regions))))
```

Comment:
Is it necessary to call list on the chained sequence? (it looks like it's only iterated over once)